prbt_moveit_config/launch/test_context.launch

```diff
@@ -19,14 +19,24 @@
 
   <!-- Load updated joint limits (override information from URDF) -->
   <group ns="$(arg robot_description)_planning">
-    <rosparam command="load" file="$(find prbt_moveit_config)/config/joint_limits.yaml"/>
+    <rosparam if="$(eval not gripper)" command="load" file="$(find prbt_moveit_config)/config/joint_limits.yaml"/>
+    <rosparam unless="$(eval not gripper)" command="load"
+              file="$(eval find('prbt_'+ gripper + '_support') + '/config/joint_limits.yaml')" />
   </group>
 
-  <!-- Load updated joint limits (override information from URDF) -->
+  <!-- Load cartesian limits -->
   <group ns="$(arg robot_description)_planning">
     <rosparam command="load" file="$(find prbt_moveit_config)/config/cartesian_limits.yaml"/>
   </group>
 
+  <!-- Load limits again in default namespace (which may not be configurable for planners) -->
```

Comment:
Why would you want to duplicate the settings?

Doesn't your test case  pilz_trajectory_generation/test/unittest_trajectory_generator_common.test already set both params by including this launch file twice with different arguments?

Reply:
yes, it is included twice and loading the robot models from different parameters/namespaces works fine, but not the initialization of the `pilz_command_planner`, due to:

> static const std::string PARAM_NAMESPACE_LIMTS = "robot_description_planning";

So I want to have all joint_limits (including gripper) in addition to be stored in the namespace `robot_description_planning`. As long as the models loaded from the urdf are discriminated correctly, this works fine.